soyjs/generator.go

```diff
@@ -8,9 +8,17 @@ import (
 	"github.com/robfig/soy/template"
 )
 
+type JSFormatter interface {
+	Template(name string) (string, string)
+	Call(name string) (string, string)
+	Directive(name string) (PrintDirective, string)
```

Comment:
seems like the second return value is never used.. can we remove?

Reply:
this is used on lines 419 and 420

Reply:
My point is that ES5Formatter and ES6Formatter always return empty string `""`, so it looks like it doesn't serve a purpose

Reply:
e.g. importString is always ""